L1Trigger/L1TGlobal/interface/ConditionEvaluation.h

```diff
@@ -342,7 +350,35 @@ namespace l1t {
                            << "\n\t passWindow2 = " << passWindow2
                            << "\n\t pass W1 || W2 = " << (passWindow1 || passWindow2) << std::endl;
 
-    if (passWindow1 || passWindow2) {
+    if (W3beginR == W2endR && W3beginR == (Type1)-1) {
```

Comment:
Does this templated check require that `Type1` be of some type with a definable subtraction with integers? Is that guaranteed?